Packages/vcs/Lib/Canvas.py

```diff
@@ -2679,11 +2679,25 @@ def plot_filledcontinents(self,slab,template_name,g_type,g_name,bg,ratio):
         except Exception,err:
             print err
 
+    def __new_elts(self,original,new):
```

Comment:
let's let it rot here then... I just can't get to it now, it fixes memory growth issue. I still need to run pep8 fix on vcs anyway, I don't want to it twice...